maintainers/scripts/update-python-libraries

```diff
@@ -169,7 +182,24 @@ def _get_latest_version_pypi(package, extension, current_version, target):
     return version, sha256
 
 
-def _get_latest_version_github(package, extension, current_version, target):
+def _get_latest_version_github(text, package, extension, current_version, target):
```

Comment:
Could you add a docstring, and explain in it how this function roughly works (that is, combination of `PyGithub` and `nix-prefetch-github`).